mgr/Conscript-standard

```diff
@@ -908,26 +908,46 @@ if ( $pkg !~ /^sim$/ && $pkg !~ /^gen$/ ) {
 	$CPPPATH = "#StarVMC" . $main::PATH_SEPARATOR . $CPPPATH;
 	#    print "StarVMC change CPPPATH to $CPPPATH =========\n";
 
-    } elsif ($pkg eq "TPCCATracker" or $pkg eq "StiCA") {
-        $CPPPATH .=
-	  $main::PATH_SEPARATOR . "#" . $Dir . $main::PATH_SEPARATOR . "#" . $Dir . "/code";
+    } elsif ( $pkg =~ m/^Vc$/){
+	#    $CPPPATH =
+	#      "#" . $Dir . $main::PATH_SEPARATOR . "#" . $Dir . "/include";
+	#    print "CPPPATH => $CPPPATH\n";
+	$CXXFLAGS .= " -W -Wall -Wimplicit -Wswitch -Wformat -Wchar-subscripts -Wparentheses -Wmultichar -Wtrigraphs -Wpointer-arith -Wcast-align -Wreturn-type -Wno-unused-function -ansi -pedantic -Wno-long-long -Wno-variadic-macros -msse -msse2 -Wshadow -fno-threadsafe-statics";
+	$CXXFLAGS .= $env->{Packages}->{Vc}->{CPP};
+#	if ($STAR_SYS =~ 'gcc432$') {$CXXFLAGS .= " -DVC_IMPL=Scalar";}
+#	else                        {$CXXFLAGS .= " -DVC_IMPL=SSE";}
+	my @srcC = ();
+	foreach my $s (@src) {
+	    #      print "$s\n";
+	    #      next if $s eq 'benchmarks/benchmark.cpp';
+	    my $dirL = File::Basename::dirname($s);
+	    my $sdir = File::Basename::basename($dirL);
+	    next if $sdir eq 'examples' or $sdir eq 'benchmarks';
+	    push @srcC, $s;
+	}
+	@src = @srcC;
 
-        if ( defined($ENV{Vc_DIR}) ) {
-          $CPPPATH = $ENV{Vc_DIR}."/include".$main::PATH_SEPARATOR.$CPPPATH;
-          $LIBPATH = $ENV{Vc_DIR}."/lib";
```

Comment:
Why are you proposing to remove these lines setting the build options for TPCCATracker and StiCA based on the `Vc_DIR` environment variable? This is related to the comment above about the hard coded `spack` command which we would like to avoid.

Reply:
Insead of reading environment variable, I defined a "package" defined
https://github.com/kehw/star-sw/blob/main/mgr/ConsDefs.pm#L1285

and use the package as
```perl
$CXXFLAGS .= $env->{Packages}->{Vc}->{CPP};
$LIBS .= $env->{Packages}->{Vc}->{LIBS};
```

Reply:
The VC package you define in ConsDefs.pm is not "instead of" the defined environment variable. It actually **uses** the same environment variable to define the package. By doing this you just introduce an extra layer between the CPPPATH and LIBPATH variables and the value set in the environment variable. While there is perhaps a deeper concept behind the Cons environments it is not clear if there is a real benefit from these extra lines of code...

Reply:
I am no cons expert. Read the file, usage like "$env->{Packages}" seems a consistent way to use third party packages. The packages was defined in ConsDefs.pm, but not used. I am trying to keep it consistent to rest of mgr/Conscript-standard.